types/remark-hint/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "DOM"
```

```suggestion
            "es6"
```
do you really need this?

---

Done.